src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -71,15 +106,27 @@ export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProp
         // todo
     };
 
-    private copyFailureDetails = (): void => {
-        // todo
+    private copyFailureDetails = (event: React.MouseEvent<any>): void => {
```

Comment:
I can't find usages of `copyFailureDetails `... it's private so only place that I should find usages is this same class but I don't see any... am I missing something?

Reply:
it's used in getMenuItems(), the copy failure item onClick

Reply:
Missed that the first time